frontend/src/screens/channels/Channels.tsx

```diff
@@ -328,6 +330,17 @@ export default function Channels() {
                     sats
                   </div>
                 )}
+                {balances && bitcoinRate && (
```

Comment:
should we create a component for displaying a sat + fiat value? I see this is duplicated, and then we could also move the hook to fetch the rates to this component

Reply:
resolved